compiler/pipes/check-classes.cpp

```diff
@@ -131,6 +132,28 @@ void CheckClassesPass::check_serialized_fields(ClassPtr klass) {
   });
 }
 
+void CheckClassesPass::check_magic_methods(ClassPtr klass) {
+  klass->members.for_each([&](const ClassMemberInstanceMethod &f) {
+    if (vk::ends_with(f.function->name, ClassData::NAME_OF_TO_STRING)) {
```

Comment:
`And because there are a lot of operations to do for `local_name()`, 
we will not slow down much because of the check for each method? 
Suffix checking should be much faster.

Reply:
Since checking magic methods has been moved to 
`FinalCheckPass`, this piece of code no longer exists.